packages/@lwc/perf-benchmarks/rollup.config.mjs

```diff
@@ -8,8 +8,7 @@
 import { fileURLToPath } from 'node:url';
 import path from 'node:path';
 import glob from 'glob';
-import replace from '@rollup/plugin-replace';
-import syntheticShadow from './rollup-plugins/synthetic-shadow.mjs';
```

Comment:
I figured rather than have a fancy Rollup plugin that injects `@lwc/synthetic-shadow` if the filename contains `ss`, we can just explicitly import it for those tests. Less magic!